### PR TITLE
Move template check in HomePageAction

### PR DESCRIPTION
**Pull request by Grynysh**
Any reason to check for template after `if ($this->template instanceof PlatesRenderer)`? For me looks like if template not added we can return JsonResponse with welcome message immediately.

### CODE REVIEW

**Comment by xtreamwayz** [comment]
@Grynysh I've changed the branch to develop since I'm not planning a new hotfix. Can you rebase it please?

**Comment by Grynysh** [comment]
Yep, done.

File: src/App/src/Action/HomePageAction.php [hunk 1]
```diff
@@ -26,6 +26,13 @@ public function __construct(Router\RouterInterface $router, Template\TemplateRen
 
     public function __invoke(ServerRequestInterface $request, ResponseInterface $response, callable $next = null)
     {
+        if (! $this->template) {
+            return new JsonResponse([
+                'welcome' => 'Congratulations! You have installed the zend-expressive skeleton application.',
+                'docsUrl' => 'zend-expressive.readthedocs.org',
```
Comment (samsonasik) [inline src/App/src/Action/HomePageAction.php]: use `docs.zendframework.com/zend-expressive` instead

Comment (webimpress) [inline src/App/src/Action/HomePageAction.php]: Or maybe even `https://docs.zendframework.com/zend-expressive/` as it was previously ..

**Comment by weierophinney** [comment]
Thanks, @Grynysh

**Pull request merged**